core/src/main/resources/hudson/logging/LogRecorder/configure.jelly

```diff
@@ -38,10 +38,10 @@ THE SOFTWARE.
     <p>
       ${%Enter the name of the logger you want to enable}.
     </p>
```

Comment:
<img width="982" alt="image" src="https://user-images.githubusercontent.com/43062514/163652556-29d46f08-f616-432a-9c1a-aae5e54bc7c1.png">

I don't think this 'Enter the name of the logger you want to enable' line is necessary, looks a bit odd with the notice in between it and the input.